dataservice/api/common/model.py

```diff
@@ -29,6 +29,7 @@ def __init__(self, *args, **kwargs):
 class IDMixin:
     """
     Defines base ID columns common on all Kids First tables
+    :param visible: Flags visibility of data from the dataservice
```

Comment:
Does it make the most sense to have this in the ID mixin?

Reply:
Moving the doc string to base model